src/routes/plans/+page.svelte

```diff
@@ -161,6 +161,13 @@
     goto(`${base}/plans/${plan.id}`);
   }
 
+  function onStartTimeChanged() {
```

Comment:
Think this may not have to do with this PR but the date inputs seem to have an internal field that only updates on valid date submission. But that means you can run into this situation (maybe we don't care) where you've entered valid dates and then entered bad values. The Create button is still enabled since the date picker fields probably still store the old valid values but it's a little confusing from the UI. Relatedly for this PR, if I choose a start time and delete the end time via keyboard (not using the clear button in the picker) the input just remembers the last date it had. Future changes to the start time don't affect the end time even though it appears blank to the user.
<img width="473" alt="image" src="https://user-images.githubusercontent.com/4419607/214450111-7cceb3c0-7856-4cb7-84b0-30eab96c105c.png">

Reply:
That's a good catch. It's definitely outside of the scope of this PR. I'll create an issue for it. Thanks!